tools/azure-sdk-tools/ci_tools/conda/conda_functions.py

```diff
@@ -79,8 +79,6 @@
     classifiers=[
         "Development Status :: 5 - Production/Stable",
         'Programming Language :: Python',
-        'Programming Language :: Python :: 2',
-        'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
```

Comment:
Just double-checking; is 3.6 also still supported by anyone?

Reply:
This is a good question. **NO** is the answer. Update inc.